repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -576,8 +576,8 @@ export const clickRandomElementInTable = (tableName: string, elementId: string):
         });
 };
 
-export const completeOperatorSearch = (isMultiService: boolean): void => {
-    clickElementById('test-radio');
+export const completeOperatorSearch = (): void => {
```

Comment:
my code from my prior pr with the fix for this function has been overwritten please take a look

Reply:
raising a fix for this myself